src/shared/utils/now.ts

```diff
@@ -0,0 +1,3 @@
+import * as dayjs from 'dayjs';
+
+export const now = dayjs().format();
```

Comment:
this is still wrong. use a function instead of direct variable assignment

Reply:
The changes here to use **dayjs** instead of **date-fns** as mentioned in commit message. I mentioned in another comment that I don't fully understand the requirement, So, will leave it until further discussion.